crawler/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN mkdir -p /app/crawler
 
 COPY ./target/crawler-0.0.1-SNAPSHOT.jar /app/crawler/crawler-0.0.1-SNAPSHOT.jar
 
-CMD java -Dspring.data.mongodb.uri=mongodb://mongodb:27017 -jar /app/crawler/crawler-0.0.1-SNAPSHOT.jar
\ No newline at end of file
+CMD java -Dspring.data.mongodb.uri=mongodb://mongodb:27017 -jar /app/crawler/crawler-0.0.1-SNAPSHOT.jar
```

Comment:
Nothing at all. Anton and I tried to notice the change but there was nothing even the spaces!!. This happened because of the some conflicts between branches. Maybe and I am not sure it is just the new line!
![image](https://user-images.githubusercontent.com/25286493/45408058-8d1c5400-b66b-11e8-8117-2112810e45af.png)